spec/features/basic_search_spec.rb

```diff
@@ -4,11 +4,19 @@
 
 RSpec.feature "Verify query search results" do
 
-  subject { Page.new }
+  # subject { Page.new }
```

Comment:
Done.
I left it actually because the inability to use subject reference anymore. So I wasn't sure whether I missed smth or this inevitable workaround. I also highlighted this in the commit description:
' -- can't invoke page instance via subject anymore, thus had to return to @page variable'